src/gt4py/next/program_processors/runners/dace_fieldview/gtir_builtin_translators.py

```diff
@@ -163,7 +163,6 @@ class PrimitiveTranslator(Protocol):
     def __call__(
         self,
         node: gtir.Node,
-        sdfg: dace.SDFG,
         state: dace.SDFGState,
```

Comment:
Yes, I could remove it. That would require a `get_current_state()` on the `SDFGBuilder` interface because a new state is added by each `Stmt`:
```
# visit one statement at a time and expand the SDFG from the current head state
for i, stmt in enumerate(node.body):
     # include `debuginfo` only for `ir.Program` and `ir.Stmt` nodes: finer granularity would be too messy
    head_state = self.sdfg.add_state_after(head_state, f"stmt_{i}")
    head_state._debuginfo = dace_utils.debug_info(stmt, default=self.sdfg.debuginfo)
    head_state = self.visit(stmt, state=head_state)
```